docs/integrations/destinations/s3-migrations.md

```diff
@@ -0,0 +1,202 @@
+|-------------------------|--------------------------|--------------------|-----------------------------------------------------------|----------------------------------------------|--------------------------------------------------------------------------------------------------------|
+| `_airbyte_ab_id`        | `_airbyte_raw_id`        | string (UUID)      | `{ "type": "string", "logicalType": "uuid" }`             | `String`                                     | Airbyte-added unique identifier.                                                                       |
+| `_airbyte_emitted_at`   | `_airbyte_extracted_at`  | integer (epoch ms) | `{ "type": "string", "logicalType": "timestamp-millis" }` | `Int64(logicalType=Timestamp(Milliseconds))` | Time at which the data was loaded from the source.                                                     |
+| [NEW FIELD]             | `_airbyte_generation_id` | integer            | `{ "type": "long" }`                                      | `Int64`                                      | Monotonically-increasing refresh id, if applicable.                                                    |
+| [NEW FIELD]             | `_airbyte_meta`          | object (see below) | Record (see below)                                        | Record (see below)                           | Additional metadata, including change data capture info and sync id.                                   |
+| `_airbyte_data`         | [UNCHANGED]              | optional object    | (from client schema)                                      | (from client schema)                         | Data payload when [flattening is disabled](https://docs.airbyte.com/integrations/destinations/s3#csv). |
+
+The `_airbyte_meta` field is an object that currently has one field:
+
+| Field Name     | Json Type | Description                                                    |
+|----------------|-----------|----------------------------------------------------------------|
+| `changes`      | list      | Change data capture info                                       |
+| `sync_id`      | integer   | Monotonically-increasing integer representing the current sync |
+
+The `changes` field is a list of objects, each of which represents a change to the data. Each object has the following fields:
+
+| Field Name | Json Type* | Description                                                                |
+|------------|------------|----------------------------------------------------------------------------|
+| `field`    | string     | The name of the field that was affected                                    |
+| `change`   | string     | The type of change (currently only `NULLED` or `TRUNCATED`)                |
+| `reason`   | string     | The reason for the change, including its origin (ie, destination-v-source) |
```

Comment:
technically, platform could also be a change origin
```suggestion
| `reason`   | string     | The reason for the change, including its origin (e.g. source/destination) |
```

Reply:
I know, I was trying to be less verbose. I added it in the s3.md though and will add it here as well.

Reply:
fixed